test.js

```diff
@@ -136,3 +136,25 @@ test('support >2e9', (t) => {
   t.ok(compare(instance.max(), recordValue), 'max match')
   t.end()
 })
+
+test('reset histogram', (t) => {
+  const instance = new Histogram(1, 100)
+  t.equal(instance.min(), 9223372036854776000, 'min is setup')
```

Comment:
Not really.
However it is a bug in the C library. We might open a separate bug there.